svelte/src/routes/v2/+page.svelte

```diff
@@ -98,7 +103,7 @@
 		
 		<RawDataInputer bind:rawData bind:doReconvert/>
 		<div class ="bg-neutral-300 h-[1px]"/>
-		<ConfigChanger/>
+		<ConfigChanger config={config}/>
```

Comment:
I thought you need to do `bind:config` here. I thought when you pass it like this, when you change the config from the ConfigChanger, the config inside +page.svelte won't be changed. need to run this to make sure

Reply:
and probably read something to correct my understanding lol

Reply:
Looks like a good idea, but no idea how to get that without errors.